scc/cert/serialization/types_test.go

```diff
@@ -0,0 +1,101 @@
+package serialization
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestHexBytes_MarshalJSON(t *testing.T) {
+	h := HexBytes([]byte{0x01, 0x2a, 0xbc})
+	data, err := h.MarshalJSON()
+	require.NoError(t, err)
+	require.Equal(t, []byte(`"0x012abc"`), data)
+	h = nil
+	data, err = h.MarshalJSON()
+	require.NoError(t, err)
+	require.Equal(t, []byte("null"), data)
+}
+
+func TestHexBytes_UnmarshalJSON_ValidHexString(t *testing.T) {
+	var h HexBytes
+	data := []byte(`"0x012abc"`)
+	require.NoError(t, h.UnmarshalJSON(data))
```

Comment:
This test should also check that h has the expected value after the unmarshalling.

Reply:
that is done in line 28